initrd/bin/kexec-unseal-key

```diff
@@ -25,22 +25,23 @@ DEBUG "CONFIG_TPM2_TOOLS: $CONFIG_TPM2_TOOLS"
 DEBUG "Show PCRs"
 DEBUG "$(pcrs)"
 
+failed=0
 for tries in 1 2 3; do
-	read -s -p "Enter LUKS Disk Unlock Key passphrase (blank to abort): " tpm_password
+	read -s -p "Enter LUKS LUKS TPM Disk Unlock Key passphrase (blank to abort): " tpm_password
```

Comment:
LUKS word repeating.

Reply:
@tlaurion Please fix this too even though I overlooked it.  (Good catch @UndeadDevel)

Reply:
@UndeadDevel thanks! @JonathonHall-Purism fixed under [4f2b1b6](https://github.com/linuxboot/heads/pull/1595/commits/4f2b1b68b0081c9e37526a8f8359d1a762a7c2ba)